packages/storybook/src/stories/playground/customize-components.stories.tsx

```diff
@@ -49,18 +60,14 @@ function LoadingComponent() {
   );
 }
 
-export function Loading() {
-  return (
-    <div className="grid-container">
-      <XGrid
-        rows={rows}
-        columns={columns}
-        components={{ loadingOverlay: LoadingComponent }}
-        loading
-      />
-    </div>
-  );
-}
+export const Loading = Template.bind({});
```

Comment:
Could we wrap that to make it more understandable on what we do here? 
Maybe 
`const getStoryTemplate = ()=> Template.bind({});`

Reply:
The following is a storybook pattern:

```
export const Foo = Bar.bind({});
Foo.args = { ... }
```

We recommend everybody use this pattern if they are using Args stories, since we have some tooling in mind that will look for this pattern in static analysis...